storage/src/add_bucket_conditional_iam_binding.php

```diff
@@ -29,20 +29,25 @@
 /**
  * Adds a conditional IAM binding to a bucket's IAM policy.
  *
- * @param string $bucketName the name of your Cloud Storage bucket.
- * @param string $role the role that will be given to members in this binding.
- * @param string[] $members the member(s) that is associated to this binding.
- * @param string $title condition's title
- * @param string $description condition's description
- * @param string $expression the condition specified in CEL expression language.
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ * @param string $role The role that will be given to members in this binding.
+ * @param string[] $members The member(s) associated with this binding.
+ * @param string $title The title of the condition.
+ * @param string $description The description of the condition.
+ * @param string $expression The condition specified in CEL expression language.
  *
  * To see how to express a condition in CEL, visit:
  * @see https://cloud.google.com/storage/docs/access-control/iam#conditions.
- *
- * @return void
  */
 function add_bucket_conditional_iam_binding($bucketName, $role, $members, $title, $description, $expression)
```

Comment:
for parameter values that are arrays, the `samples_helper.php` function needs a typehint so it can explode user-input:
```suggestion
function add_bucket_conditional_iam_binding($bucketName, $role, array $members, $title, $description, $expression)
```

Otherwise a string will be passed in on the CLI, and that will result in an error somewhere down the line, I assume.